eventmesh-webhook/eventmesh-webhook-admin/src/main/java/org/apache/eventmesh/webhook/admin/FileWebHookConfigOperation.java

```diff
@@ -140,12 +117,50 @@ private WebHookConfig getWebHookConfigFromFile(File webhookConfigFile) {
 			while ((line = br.readLine()) != null) {
 				fileContent += line;
 			}
-		} catch (FileNotFoundException e) {
-			e.printStackTrace();
 		} catch (IOException e) {
-			e.printStackTrace();
+			logger.error("get webhook from file {} error", webhookConfigFile.getPath(), e);
 		}
 		return JsonUtils.deserialize(fileContent, WebHookConfig.class);
 	}
 
+	private synchronized boolean writeToFile(File webhookConfigFile, WebHookConfig webHookConfig) {
+		FileOutputStream fos = null;
+		BufferedWriter bw = null;
+		FileLock lock = null;
```

Comment:
Please use try with resource, this can help you to close the resource in a safe way, of you need to add multiple try catch in your finally block.
```java
try{
    if (fos != null) {
		fos.close();
	}
}catch (IOException e){
}
try{
    if (bw != null) {
		bw.close();
	}
}catch (IOException e){
}
try{
   if (lock != null) {
		lock.close();
	}
}catch (IOException e){
}